manager/lamptest/lamptest.cpp

```diff
@@ -159,10 +159,11 @@ void LampTest::storePhysicalLEDsStates()
         }
 
         phosphor::led::Layout::Action action = getActionFromString(state);
-        if (action != phosphor::led::Layout::Off)
+        if (action != phosphor::led::Layout::Action::Off)
```

Comment:
could have removed phosphor::led:: so that code is same like above

Reply:
Done